### ISSUE TITLE
Bug: afterSendSingleMessage callback trigger when typing

### ISSUE THREAD

**Issue by AndrewZuo01** [openimsdk/open-im-server#1701]
### What happened?

after send single message callback trigger when typing

### What did you expect to happen?

I shouldn't trigger when typing

### How can we reproduce it (as minimally and precisely as possible)?

open afterSendSingleMessage callback, and type message

### version

<details>
```console
$ {name} version
# paste output here
```
</details>

### Cloud provider

<details>
</details>

### OS version

<details>
```console
# On Linux:
$ cat /etc/os-release
# paste output here
$ uname -a
# paste output here
# On Windows:
C:\> wmic os get Caption, Version, BuildNumber, OSArchitecture
# paste output here
```
</details>

### Install tools

<details>
</details>

**Comment by kubbot** [comment]
This issue is available for anyone to work on. **Make sure to reference this issue in your pull request.** :sparkles: Thank you for your contribution! :sparkles:
[Join slack 🤖](https://join.slack.com/t/openimsdk/shared_invite/zt-22720d66b-o_FvKxMTGXtcnnnHiMqe9Q) to connect and communicate with our developers.
If you wish to accept this assignment, please leave a comment in the comments section: `/accept`.🎯

**Comment by cubxxw** [comment]
Thank you for your support. Your assistance is greatly appreciated if you have any questions.

**Comment by yasyx** [comment]
I haven't been able to reproduce the exact issue described in the problem, but I have indeed encountered some unexplained callback behavior related to callbackAfterSendSingleMsgCommand. In one case, I received two consecutive callbacks when sending a message. In another case, I'm unsure of the cause as I confirmed that I didn't perform any active actions to send a message when receiving the callback.

If this is not the expected behavior, I will attempt to fix it.
<img width="1773" alt="image" src="https://github.com/openimsdk/open-im-server/assets/12021357/41d8a3b8-0333-4ec1-a527-aac2a54e1b34">

**Comment by yasyx** [comment]
This issue might have already been fixed. As for the other callbacks, I checked the contentType, and they mainly consist of callbacks for user-read messages and user status change messages. These callbacks cannot be considered unreasonable. I believe it's necessary to consider the actual business scenarios to determine which callbacks are truly needed.
![image](https://github.com/openimsdk/open-im-server/assets/12021357/892f8e1a-24e6-4845-977a-f6fd0bf9c6ab)
![image](https://github.com/openimsdk/open-im-server/assets/12021357/70a48403-fe3a-4780-81ea-cd40b69ec3ac)
